flyway-core/src/main/resources/org/flywaydb/core/internal/dbsupport/greenplum/upgradeMetaDataTable.sql

```diff
@@ -0,0 +1,21 @@
+--
+-- Copyright 2010-2016 Boxfuse GmbH
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--         http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+
+ALTER TABLE "${schema}"."${table}" DROP COLUMN "version_rank";
```

Comment:
This file is not necessary. It is only there to allow Flyway 3.x metadata tables to be migrated to the 4.0 format.